cmd/oceanbench/monitor.go

```diff
@@ -64,15 +65,16 @@ type sensorData struct {
 
 // monitorDevice holds the relevant information for each device.
 type monitorDevice struct {
-	Device     model.Device
-	Address    string
-	Sending    string
-	StatusText string
-	Uptime     string
-	Count      int // Number of scalars sent in the monitor period.
-	MaxCount   int // Max number of scalars that could be sent.
-	Throughput int // Percentage of successful scalars.
-	Sensors    []sensorData
+	Device       model.Device
+	Address      string
+	Sending      string
+	StatusText   string
+	Uptime       string
+	LastReported string
```

Comment:
I kind of think this should be a ```Time```

Reply:
This is only used to populate the template. Having it as a string means that we can control the formatting better. Unless there's a better way of doing this in the template?

Reply:
You can have a method that formats it and call that in the template :nerd_face:

Reply:
I have used unix timestamps instead, as these are already compatible with our exported template methods (localdatetime)